modules/jupyter-widget/src/utils.js

```diff
@@ -24,7 +24,18 @@ export function updateDeck(inputJSON, {jsonConverter, deckgl}) {
   deckgl.setProps(results);
 }
 
-export function initDeck({mapboxApiKey, container, jsonInput, tooltip, onComplete, handleClick}) {
+export function initDeck(
+  {mapboxApiKey, container, jsonInput, tooltip, onComplete, handleClick, handleError} = {
+    mapboxApiKey: null,
+    container: null,
+    jsonInput: null,
+    tooltip: null,
+    onComplete: null,
+    handleClick: null,
+    // eslint-disable-next-line
+    handleError: console.error
```

Comment:
`onError: console.error.bind(console)`?

Usually you need to do `console.error.bind(console)`, as the methods cannot be called without an object (it might work if some other library has already redefined global console, but better not bet on that).